### ISSUE TITLE
The results of ResNeXt-50 (32x4d) on ImageNet

### ISSUE THREAD

**Issue by xieyxclack** [bearpaw/pytorch-classification#51]
Many thanks for your valuable work!

I have trained ResNeXt-50 (32x4d) on ImageNet following the training recipes,  and I got a 75% valid accuracy (the reported result is 77%). 

I have not modified any code and configs, except I train it on 8xGPUs with torch.nn.DataParallel. Do you have any hints about the performance drop?

Thanks again for your work, and I hope to hear some insights from you!
